package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Rentzila",
+  "type": "module",
+  "scripts": {
+    "wdio-all": "npx wdio run ./wdio.conf.ts"
+  },
+  "devDependencies": {
+    "@wdio/allure-reporter": "*",
```

Comment:
@Anastas11aZakharova I'm afraid, that this is not a good idea. May be for yours repo this is fine, but normally, the versions must be stable, if you leave them like that the versions could vary for different local machines depends on when the repo has been cloned, or, even worse, the one of the versions of the dependencies will cause issues, and the CI will fail. In short - add stable version of the dendecies and the node version recommended must be added to the README file.

Reply:
Fixed